app/policies/feed_policy.rb

```diff
@@ -0,0 +1,110 @@
+class FeedPolicy < ApplicationPolicy
+  class Scope < ApplicationPolicy::Scope
+    # All users (including anonymous) can see all feeds
+    def resolve
+      scope.all
+    end
+  end
+
+  def index?
+    # Everyone can view the list
+    true
+  end
+
+  def show?
+    # Everyone can view individual feeds
+    true
+  end
+
+  def new?
+    super || can_create_new_feed?
+  end
+
+  def create?
+    super || can_create_feed?
+  end
+
+  def edit?
+    super || can_edit_feed?
+  end
+
+  def update?
+    super || can_update_feed?
+  end
+
+  def destroy?
+    super || can_destroy_feed?
+  end
+
+  private
+
+  # Feeds can be created by any admin of any group
+  def can_create_new_feed?
+    return false unless user
+    user.admin_groups.any?
+  end
+
+  # Feeds can be created by any admin of the associated group
+  # NOTE: This assumes Feed will have a group association in the future
+  def can_create_feed?
+    return false unless user
+    return false unless record.respond_to?(:group) && record.group.present?
```

Comment:
If feeds don't have a group association yet, let's create an issue to track this so we can tackle it and the policy work in a future PR.